### PR TITLE
Add addToRecipesToCook method and some sad path tests in user-test

### PR DESCRIPTION
**Pull request by sme93**
# Pull Request Template
## Description
Please include a summary of the change and which issue is fixed. Add recipesToCook/addToRecipesToCook method and tests, sad path testing in User and User Class

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring
## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: test/user-test.js [hunk 1]
```diff
@@ -5,71 +5,54 @@ import recipeData from '../src/data/recipes.js'
 
 let id;
 let username;
-let pantry;
 
 describe('User', () => {
   beforeEach(() => {
     id = 2;
     username = 'Ruth';
-    pantry = [{
```
Comment (mcraig2342) [inline test/user-test.js:14]: Couldn't we just add const user = new User(id, username) to the before each so we don't have to write it before every test. I know you hate before each lol but I think it makes a lot of sense in this case

File: src/user.js [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
   removeFromFavorites(recipe) {
     const i = this.favoriteRecipes.indexOf(recipe);
-    this.favoriteRecipes.splice(i, 1)
+    this.favoriteRecipes.splice(i, 1);
+  }
+
+  addToRecipesToCook(recipe) {
+    if (!this.recipesToCook.includes(recipe)) {
+      this.recipesToCook.push(recipe);
+    }
```
Comment (mcraig2342) [inline src/user.js:23]: looks good!

**Review by mcraig2342 — CHANGES_REQUESTED** [review]
The only change I would make is adding that new User instantiation into the before each, other than that it looks good!

**Review by sarahlane8 — APPROVED** [review]
Looks great! Like the syntax changes.  
I see that you took off the pantry property on the user.  We may want to consider adding it back in later, when we see how the Pantry Class develops.  We will have to see!

**Pull request merged**